network-api/networkapi/wagtailpages/migrations/0066_merge_20190404_1937.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
What does this migration achieve? O_o

Reply:
I think this happened when I pulled PR https://github.com/mozilla/foundation.mozilla.org/pull/2962 code into this branch (we did this to test out the POST functionality). This branch was branched off your `signup-bcpage` branch and `signup-bcpage` was out of date with `master` at the time and hence resulted in migration issue.

Reply:
Should we have `signup-bcpage` updated first?

Reply:
Ah. No, for migrations you can run as many migrations during local dev as you like, but when you rebase you need to delete every migration file that got built as part of the PR, and regenerate a single one instead, using `inv makemigrations`

Reply:
Um okay. I don't know what happened then. I don't remember my code needed `inv makemigration`... 

Anyways I'll delete this file.